spec/support/database_setup.rb

```diff
@@ -60,19 +62,20 @@ def down
       end
     end
 
-    commentable_model = proc do |table_name|
+    commentable_model = proc do |table_name, &block|
       Class.new(ActiveRecord::Base) do
         self.table_name = table_name
         scope :random, -> { offset(rand(count)).first }
         has_many :comments, as: :commentable
+        instance_eval(&block) if block
```

Comment:
Very cool! With this new instance eval, we can also drop `commentable_model` at all and keep all being declared as `model`.